packages/health-gorilla-react/src/useHealthGorillaLabOrder.test.tsx

```diff
@@ -214,6 +231,33 @@ describe('useHealthGorilla', () => {
     });
   });
 
+  test.each([
+    { resourceType: 'Location', id: 'L-123' } satisfies Location,
+    { reference: 'Location/L-123' } satisfies Reference<Location>,
+    { resourceType: 'Organization', id: 'O-123' } satisfies Organization,
+    { reference: 'Organization/O-123' } satisfies Reference<Organization>,
+  ])(
+    'Requesting Location set',
+    async (
+      requestingLocation: Location | Organization | (Reference<Location | Organization> & { reference: string })
+    ) => {
+      const { result } = setup({ patient, requester, requestingLocation });
+      // Make sure the order is valid
+      await act(async () => {
+        result.current.addTest(RWS_AOE_TEST);
+        result.current.setPerformingLab({ resourceType: 'Organization', id: 'Lab-123' });
+        result.current.updateBillingInformation({ billTo: 'patient' });
+      });
+
+      const { serviceRequest } = await result.current.createOrderBundle();
+
+      const expectedValue = isReference(requestingLocation) ? requestingLocation : createReference(requestingLocation);
```

Comment:
I would suggest moving the expected value as a second field in the `test.each` array, but that might just be a personal preference of mine; not sure which is "better".

```typescript
test.each([
  [{ resourceType: 'Location', id: 'L-123' }, { reference: 'Location/L-123'}],
  ...
])('Requesting Location set', async (requestingLocation, expectedRef) => {
  ...
```